src/main/java/mat/server/FeatureFlagServiceImpl.java

```diff
@@ -8,17 +8,19 @@
 import mat.dao.FeatureFlagDAO;
 import mat.model.FeatureFlag;
 
+import java.util.List;
+
 @Service
 public class FeatureFlagServiceImpl extends SpringRemoteServiceServlet implements FeatureFlagService {
 
 	private static final long serialVersionUID = 1L;
 
 	@Autowired
 	FeatureFlagDAO featureFlagDAO;
-	
+
 	@Override
-	public FeatureFlag findFeatureFlag(String flagName) {
-		return featureFlagDAO.findFlagByName(flagName);
+	public List<FeatureFlag> findFeatureFlag() {
```

Comment:
it probably makes sense to give it a plural name, like findFeagureFlags or findAllFeatureFlags or something similar

Reply:
Noted